cpp.md

```diff
@@ -168,6 +168,28 @@ default:
  //code to be executed when all the above cases are not matched;    
 } 
 ```
+## Break and Continue Statements
+
+### 1. Break
+```c
+for(i=0; i<5; i++){
+   if(i==4){
+      break;
+      }
+}
+```
+The break statement helps in coming out from the loop when your condition is satisfied.
+
+### 2. Continue
+```c
+for(i=0; i<5; i++){
+   if(i==4){
+      continue;
```

Comment:
can you add more realistic example

Reply:
Can you please suggest me something like should i display the output of the code as well to explain it better or if any other example of code you think which can explain continue statement better ?

Reply:
@arshanand1  i think you can also include comments in your code it will make your code understandable for the people

Reply:
@kshitijmishra27 For sure! Thanks for the suggestion.

Reply:
1. Keep it precise 
2. Add a real-world examples i.e where break, continue are really useful and you can't solve without them

Reply:
Real life examples added to the file cpp.md